test/banach/retry_test.clj

```diff
@@ -5,6 +5,40 @@
             [clojure.math.numeric-tower :as math]
             [banach.retry :as retry]))
 
+(def ctx-1 {:failures [(Exception. "Slow your roll")]})
+(def ctx-2 (update ctx-1 :failures conj (Exception. "KABOOM")))
+
+(deftest routing-tests
+  (let [strat (retry/routing
+               (comp odd? count :failures) identity
+               (constantly true) retry/give-up)]
```

Comment:
Oh, wait, I see your point. The confusion isn't about fallbacks, it's that `give-up` is your fallback! That's a good point.